pyls/plugins/preload_imports.py

```diff
@@ -30,5 +30,8 @@ def pyls_initialize(config):
         try:
             __import__(mod_name)
             log.debug("Preloaded module %s", mod_name)
-        except ImportError:
+        except Exception:
```

Comment:
Now Pylint is complaining about this, sorry. Please add a comment to avoid that.